### PR TITLE
[READY]OreshkoJA lab1

### PR DESCRIPTION
**Pull request by OreshkoJA**

### CODE REVIEW

**Comment by OreshkoJA** [comment]
Did i make the right branch?

**Comment by Toxa-man** [comment]
> Did i make the right branch?

yep

File: OreshkoJA/lab1 [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+			break;
+		case '-': result = first - second;
+			break;
+		case '*': result = first * second;
+			break;
+		case '/': result = first / second;
+			break;
+		default: cout << "Неправильный ввод знака" << endl;	
+			system("pause");
+			return 0;
```
Comment (Toxa-man) [inline OreshkoJA/lab1]: That place doesn't fixed since previous pull request

**Comment by OreshkoJA** [comment]
I hope it's ok now

**Comment by Toxa-man** [comment]
> I hope it's ok now

Not exactly
If i got a zero divison error, i still can see the "result" message, however i should not

**Comment by OreshkoJA** [comment]
hmmm..
Something like this?

**Comment by OreshkoJA** [comment]
I think it's so tedious to check programs fo several time...
So thank you for your patience

**Comment by Toxa-man** [comment]
>hmmm..
>Something like this?

Sorry, but still not
1) condition like if (sign == '+', '-', '/', '*') doesn't work as you may expect, as the comma operator has absolutly different side effect. Running ahead, it should look more like (sign == '+' || sign == '-' || and so on).
But it doesn't really good way to solve the problem as we perfom checking sign variable twice, First - in swtich statement and second - within if.
Little hint: why not to save out "state" of program in the separated variable and the just check it to know, do we got error or not?

**Comment by OreshkoJA** [comment]
Eeeey

**Pull request merged**